app/views/checklist/_action_list.html.erb

```diff
@@ -1,4 +1,5 @@
 <% items.each.with_index do | action, index | %>
+  <% action_index = "#{audience_index}.#{'%02d' % (index + 1)}"%>
```

Comment:
Any chance this could go over 99? I assume you'll hit the same ordering problem if you had 1.99 then 1.100 but I'm guessing this risk is rather low.

Reply:
> Code looks good to me, any concerns about this not being backwards compatible with the previous data?

Performance analytics are happy with the change and are aware that some data might be out of order.

Regarding the 1.100 - I would like to hope that we don't ever get to a situation where there would be a 100 action as a finder would be a better use case for that scenario. If you think we should still make the change, happy to do so.

Reply:
I don't have enough context to know how crazy it would be for us to hit 100, so happy with a max of 99 if you're confident that is sufficient 👍

Reply:
I feel fairly confident that we won’t hit that, currently we haven’t hit 20 yet. This is also per category so should be fine.

Reply:
Spoke to the Performance analysis team, they don't think it will hit 100.